src/gateway/kong-production/monitoring/statsd.md

```diff
@@ -1,6 +1,99 @@
+observe the collected monitoring data. 
+
+### Prerequisites
+* [Docker](https://docs.docker.com/get-docker/) is installed locally. It is used to run {{site.base_gateway}}, the supporting database, and StatsD locally. 
+
+   If you will be using an existing {{site.base_gateway}} instance instead of the test instance, you may need configure Docker to ensure features like network connectivity function correctly.  
+* [curl](https://curl.se/) is installed locally. It is used to send requests to {{site.base_gateway}}. `curl` is pre-installed on most systems.
+* [Netcat](http://netcat.sourceforge.net/) is installed locally as `nc` on the `PATH`. `nc` is used to send requests 
+  to the StatsD management interface. `nc` is pre-installed on many systems.
+
+### Configure StatsD monitoring
+
+{:.note}
+   > **Note:** This guide's instructions use a test {{site.base_gateway}} instance to demonstrate how the StatsD plugin can be used to collect metrics. If you want to use an existing {{site.base_gateway}} instance for this guide, you must modify the connection information in the commands.
+
+1. Optional: Install {{site.base_gateway}} using the following script:
+   ```sh
+   curl -Ls get.konghq.com/quickstart | sh -s -- -m
+   ```
+   The `-m` flag instructs the script to install a mock service that is used in this guide to generate sample metrics.
+
+1. Run a StatsD container to capture monitoring data:
+
+   ```sh
+   docker run -d --rm -p 8126:8126 \
+     --name kong-quickstart-statsd --network=kong-quickstart-net \
+     statsd/statsd:latest
+   ```
+
+1. Install the [StatsD {{site.base_gateway}} plugin](/hub/kong-inc/statsd/), 
+   configuring the hostname and port of the listening StatsD service:
+
+   ```sh
+   curl -X POST http://localhost:8001/plugins/ \
+       --data "name=statsd"  \
+       --data "config.host=kong-quickstart-statsd" \
+       --data "config.port=8125"
+   ```
+
+   You should receive a JSON response with the details of the installed plugin.
+
+1. Generate sample traffic to the mock service. This allows you to observe 
+   metrics generated from the StatsD plugin. The following command generates 60 
+   requests over one minute. Run the following in a new terminal:
+
+   ```sh
+   for _ in {1..60}; do {curl localhost:8000/mock/request; sleep 1; } done
+   ```
+
+1. Query the StatsD management interface to see the collected metrics from {{site.base_gateway}}:
+
+   ```sh
+   echo "counters" | nc localhost 8126
+   ```
+
+   You should see a response similar to the following:
+
+   ```text
+   {
+     'statsd.bad_lines_seen': 0,
+     'statsd.packets_received': 56,
+     'statsd.metrics_received': 56,
+     'kong.mock.request.count': 7,
+     'kong.mock.request.status.200': 7,
+     'kong.mock.request.status.total': 7
+   }
+   END
+   ```
+
+See the [StatsD plugin](/hub/kong-inc/statsd/) 
+documentation for more information about how to use and configure the plugin.
+
+### Clean up the test instance 
```

Comment:
@rspurgeon Would this heading work for you? I wanted to add more to "clean up" just so users can know what the section will cover at a glance.

Reply:
@cloudjumpercat I pushed a revision, please take a look.  I personally think just "Clean up" is sufficient, but if you want additional text "test instance" isn't exactly accurate, because the instructions inform the user to cleanup the test kong gateway, database and network along with the StatsD service.  I modified to be vague and say "running software"